src/main/scala/rocket/ICache.scala

```diff
@@ -164,6 +174,7 @@ class ICacheModule(outer: ICache) extends LazyModuleImp(outer)
   }
 
   val s1_tag_disparity = Wire(Vec(nWays, Bool()))
+  val s1_tlError = Wire(Vec(nWays, Bool()))
```

Comment:
The casing of this variable name makes me want to cry. Personally I'm in favor of continuing the convention of `naming_wires_with_underscores` as opposed to `namingAbstractScalaObjectsWithCamelCase`. If you really want to switch everything to camel case then fine, but you really can't have both in one name XD